Casks/b/balsamiq-wireframes.rb

```diff
@@ -9,7 +9,16 @@
 
   livecheck do
     url "https://builds.balsamiq.com/bwd/mac.jsonp"
-    regex(/"version"\s*:\s*"(\d+(?:\.\d+)+)"/i)
+    regex(/callback\((.+?)\);/i)
```

Comment:
```suggestion
    regex(/callback\((.+)\)/i)
```

The semicolon may be optional in this context, so there may be something to be said for omitting it.

This is one case where a greedy wildcard match (`.+` instead of `.+?`) makes sense, as a non-greedy match would stop once it reaches the first closing parenthesis. Our intention is to match the entire argument, so using a greedy match here will sidestep any concerns about `)` appearing in the argument text and causing an incomplete match.

Reply:
Thanks, it's a force of habit at this point. 😄